Cargo.toml

```diff
@@ -31,7 +31,7 @@ h2 = "0.2.1"
 itoa = "0.4.1"
 log = "0.4"
 pin-project = "0.4"
-time = "0.1"
+time = { version = "0.2.7", default-features = false, features = ["std"] }
```

Comment:
There's no particular reason to require 0.2.7. 0.2.4 would make the most sense, given it lowered the MSRV to 1.34 (from 1.40 beforehand).

Reply:
make sense